package.json

```diff
@@ -69,7 +72,7 @@
     "node-spawn-server": "1.0.1",
     "nopt": "4.0.1",
     "numeral": "2.0.6",
-    "plywood": "0.17.26",
+    "plywood": "^0.18.0",
```

Comment:
Could you pin version for build reproducibility? I don't trust sem versioning of NPM packages.